client/components/Omnichannel/modals/CloseChatModal.js

```diff
@@ -57,13 +67,18 @@ const CloseChatModal = ({ onCancel, onConfirm, ...props }) => {
 					</Field.Row>
 					<Field.Error>{commentError}</Field.Error>
 				</Field>
-				{Tags && (
+				{Tags && tagsList && tagsList.length > 0 ? (
 					<Field>
 						<Field.Label mb='x4'>{t('Tags')}</Field.Label>
 						<Field.Row>
-							<Tags value={tags} handler={handleTags} />
+							<Tags value={Object.values(tags)} handler={handleTags} />
 						</Field.Row>
 					</Field>
+				) : (
+					<Field>
+						<Field.Label mb='x4'>{t('Tags')}</Field.Label>
+						<TagsManual tags={tags} handler={handleTags} />
```

Comment:
IMO we should have just one component, `Tags`, and deal with the different behaviors internally rather than importing multiple components across the code base.